xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -109,17 +156,20 @@ type ServerConfig struct {
 // content. It doesn't cover NodeProto because NodeProto isn't used by
 // federation.
 func (sc *ServerConfig) String() string {
-	var ver = "xDSv3"
-	return strings.Join([]string{sc.ServerURI, sc.CredsType, ver}, "-")
+	features := ""
+	if len(sc.ServerFeatures) != 0 {
+		features = strings.Join(sc.ServerFeatures, "-")
+	}
```

Comment:
Since `strings.Join` returns `""` when given an empty slice, this can be simplified to:

```go
features := strings.Join(sc.ServerFeatures, "-")
```

Reply:
Done, thanks.